openid-4-verifiable-presentations-1_0.md

```diff
@@ -2001,68 +2028,263 @@ Note: The `nonce` and `aud` are set to the `nonce` of the request and the Client
 
 # IANA Considerations
 
-## Response Types
+## OAuth Authorization Endpoint Response Types Registry
+
+This specification registers the following `response_type` values
+in the IANA "OAuth Authorization Endpoint Response Types" registry [@IANA.OAuth.Parameters]
+established by [@!RFC6749].
+
+### vp_token
 
 * Response Type Name: `vp_token`
 * Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
-* Specification Document(s): https://openid.net/specs/openid-4-verifiable-presentations-1_0.html
+* Specification Document(s): (#response) of this specification
+
+### vp_token id_token
 
 * Response Type Name: `vp_token id_token`
 * Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
-* Specification Document(s): https://openid.net/specs/openid-4-verifiable-presentations-1_0.html
+* Specification Document(s): (#response) of this specification
 
-Note: Plan to register the following Response Types in the [OAuth Authorization Endpoint Response Types IANA Registry](https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml#endpoint).
+## OAuth Parameters Registry
 
-## Media Types
-### application/verifier-attestation+jwt {#va_media_type}
+This specification registers the following OAuth parameters
+in the IANA "OAuth Parameters" registry [@IANA.OAuth.Parameters]
+established by [@!RFC6749].
+
+### presentation_definition
+
+* Name: presentation_definition
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
```

Comment:
this does not sound right. I know OpenID4VP spec officially sits in Connect WG now, but once it goes final, it gets moved to DCP WG. Practically, this should be DCP WG

Reply:
Would there be any issue with the Connect WG appointing DCP WG as the change controller from the start? I couldn't find any rules that require otherwise.

Reply:
It would be reasonable to ask the Connect WG to make that decision.

Reply:
Agree with @jogu if its possible for the Connect WG to designate the DCP as the change controller here that would prevent a re-work here and is in practise where all the expertise lies in order to perform the change control function.

Reply:
I suggest that one of the editors send a request to openid-specs-ab@lists.openid.net requesting consensus for this change to the spec.

Reply:
This was discussed briefly on today's WG call - as our plan is now to go to an implementer's draft and move VP to the DCP WG prior to going to final / requesting IANA registrations it's fine to just leave this as connect WG for now and update it when the spec is adopted in DCP WG. I'll hence mark this conversation as resolved.